packages/design-system/src/scss/variables/colors.scss

```diff
@@ -14,6 +15,7 @@ $font-primary-color: rgba(66, 77, 87, 1);
 $font-secondary-color: rgba(66, 77, 87, 0.7);
 $font-tertiary-color: rgba(66, 77, 87, 0.5);
 $font-placeholder-color: rgba(66, 77, 87, 0.4);
+$font-primary-blue-color: $primary-color;
```

Comment:
Let's not complicate it now. Soon we will have a new color tokens

Reply:
fixed in d51a706